sycl/include/sycl/ext/oneapi/experimental/builtins.hpp

```diff
@@ -16,6 +16,8 @@
 
 #include <CL/__spirv/spirv_ops.hpp>
 
+#include "bfloat16.hpp"
```

Comment:
```suggestion
#include <sycl/ext/oneapi/experimental/bfloat16.hpp>
```

Use full path following other files usage.